spec.html

```diff
@@ -4393,6 +4393,21 @@ <h1>The Lexical Environment and Environment Record Specification Types</h1>
       <p>The Lexical Environment and Environment Record types are used to explain the behaviour of name resolution in nested functions and blocks. These types and the operations upon them are defined in <emu-xref href="#sec-lexical-environments"></emu-xref>.</p>
     </emu-clause>
 
+    <emu-clause id="sec-abstract-closure">
+      <h1>The Abstract Closure Specification Type</h1>
+      <p>The <dfn>abstract closure</dfn> specification type is used to refer to algorithmic steps together with a collection of values. Abstract closures are meta-values and are invoked using function application style such as _closure_(_arg1_, _arg2_). Like abstract operations, invocations perform the algorithmic steps described by the abstract closure.</p>
+      <p>In algorithmic steps that create an abstract closure, values are captured with the verb "capture" followed by a list of aliases. The value associated with each alias at that time is captured. In algorithmic steps describing calling an abstract closure, captured values are referred to by the alias that was used to capture the value.</p>
```

Comment:
- The spec says "algorithm steps" rather than "algorithmic steps".
- The antecedent for "at that time" is unclear.
- The phrase "algorithmic steps describing calling an abstract closure" sounds like steps that call the closure, which is presumably not what you mean.

```suggestion
      <p>In algorithm steps that create an abstract closure, values are captured with the verb "capture" followed by a list of aliases. When the abstract closure is created, it captures the value of each such alias at that time. In the algorithm steps that constitute the definition of an abstract closure, each captured value is referred to by the alias that was used to capture the value.</p>
```

Reply:
I don't agree that "constitute the definition" is any clearer.

Reply:
I think it improves the clarity a bit. Overall, I prefer @jmdyck's suggestion to what I wrote.

Reply:
We could maybe change:
```
In the algorithm steps that constitute the definition of an abstract closure
```
to:
```
In the algorithm steps of the abstract closure
```
or just:
```
In the abstract closure's algorithm steps
```

Reply:
I do like "In the abstract closure's algorithm steps", will adopt that.